src/index.css

```diff
@@ -1,13 +1,332 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;800&family=Roboto+Slab:wght@100;200&family=Silkscreen:wght@400;700&display=swap');
+
 body {
+  background-color: #0554f2;
+  background-image: url(https://wallpapercave.com/wp/wp2595119.jpg);
+  color: #fff;
+  width: 100%;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.main-continer {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! You can use this `design` once your project is approved because you need to complete this project using the [Behance design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) provided by Microverse. I am sorry to ask you to change this. Please, I request you to work more on the design of this project in the Mobile version, because the main content of this project is in the Mobile version. I really appreciate your hard work!

![image](https://user-images.githubusercontent.com/63915024/188012986-f13b7d7c-8e8d-41b1-8a03-4e5ce2379f9b.png)